common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/OneForOneBlockFetcher.java

```diff
@@ -142,7 +142,8 @@ private AbstractFetchShuffleBlocks createFetchShuffleMsgAndBuildBlockIds(
       boolean areMergedChunks) {
     String[] firstBlock = splitBlockId(blockIds[0]);
     int shuffleId = Integer.parseInt(firstBlock[1]);
-    boolean batchFetchEnabled = firstBlock.length == 5;
+    int shuffleSequenceId = Integer.parseInt(firstBlock[2]);
```

Comment:
note to reviewers: still working on this one, need to be fixed. So please ignore this one.